### PR TITLE
A little cleanup

### PR DESCRIPTION
**Pull request by cznewt**

### CODE REVIEW

**Comment by pupapaik** [comment]
Did you test following changes? and can you push single commit?

File: etcd/server.sls [hunk 1]
```diff
@@ -4,12 +4,8 @@
 etcd_packages:
   pkg.installed:
   - names: {{ server.pkgs }}
-{%- if server.get('engine', 'systemd') %}
-  - require:
-    - file: /etc/default/etcd
-{%- endif %}
 
-{%- if server.get('engine', 'systemd') == 'kubernetes' %}
+{%- if server.engine == 'kubernetes' %}
```
Comment (pupapaik) [inline etcd/server.sls:8]: This should be there. Remove this from map.jinja. Default is container.

File: README.rst [hunk 2]
```diff
@@ -57,16 +57,10 @@ ETCD proxy
         bind:
           host: 10.0.175.101
         proxy: true
-        members:
```
Comment (pupapaik) [inline README.rst:60]: Why did you delete this?

File: etcd/map.jinja [hunk 3]
```diff
@@ -1,7 +1,10 @@
-{% set server = salt['grains.filter_by']({
+
+{%- set server = salt['grains.filter_by']({
     'Debian': {
         'pkgs': ['etcd', 'python-etcd'],
-        'services': ['etcd']
+        'service': etcd,
+        'proxy': False,
+        'engine': 'systemd'
```
Comment (pupapaik) [inline etcd/map.jinja:7]: systemd is not default.

**Review by pupapaik — CHANGES_REQUESTED** [review]
Please check my comments.

File: etcd/server.sls (hunk 1 above)
Comment (fpytloun) [reply etcd/server.sls:8]: Defaults should be defined in map.jinja or using `get('xyz', default)` or `|default(default)` jinja filter.

File: etcd/server.sls [hunk 4]
```diff
@@ -24,27 +20,33 @@ etcd_service:
...
 /etc/kubernetes/manifests/etcd.manifest:
   file.managed:
-    - source: salt://etcd/files/etcd.manifest
-    - template: jinja
-    - user: root
-    - group: root
-    - mode: 644
-    - makedirs: true
-    - dir_mode: 755
+  - source: salt://etcd/files/etcd.manifest
```
Comment (fpytloun) [inline etcd/server.sls:23]: I preffer 2-space indent for lists which is IMHO correct YAML approach, while both are working fine.
See examples here, all are indented:
https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html

File: etcd/server.sls [hunk 5]
```diff
@@ -24,27 +20,33 @@ etcd_service:
...
+  - template: jinja
+  - user: root
```
Comment (fpytloun) [inline etcd/server.sls:25]: Maybe don't define defaults (user/group, mode) as it's useless

File: etcd/server.sls (hunk 4 above)
Comment (fpytloun) [reply etcd/server.sls:23]: And it's wrong by specs!
http://yaml.org/spec/1.2/spec.html

> Example 8.3. Invalid Block Scalar Indentation Indicators
> 
> ```
> ERROR:
> - A leading all-space line must
>   not have too many spaces.
> - A following text line must
>   not be less indented.
> - The text is less indented
>   than the indicated level.
> ```

**Review by fpytloun — CHANGES_REQUESTED** [review]

**Comment by fpytloun** [comment]
Old review with conflicts, closing

**Pull request closed without merging**